docs/integrations/sources/mailchimp-migrations.md

```diff
@@ -1,5 +1,30 @@
 # Mailchimp Migration Guide
 
+## Upgrading to 2.0.0
+
+### Refresh affected schemas and reset data
```

Comment:
Can we be explicit as to why we need a reset?

Reply:
updated